api-examples/core/link/statement.api

```diff
@@ -0,0 +1,8 @@
+With Class "Article" {
+    // Follow the link and run top-level steps
+    Expect Link "author"
```

Comment:
@bergos this is related to what you mentioned some time ago

Would you rather see `Expect Link` or maybe `Follow Link "author"` is more understandable? 

I also intend to have a `Follow` keyword used in other context, so I guess that makes sense...

Reply:
Decided to allow both `Expect` and `Follow` where it makes sense, FYI